modular_skyrat/modules/modular_items/lewd_items/code/lewd_helpers/species.dm

```diff
@@ -0,0 +1,74 @@
+/datum/species/proc/throw_arousalalert(level, atom/movable/screen/alert/aroused_x/arousal_alert, mob/living/carbon/human/targeted_human)
```

Comment:
Will need fixing wherever this is called, also all of these need at least a tiny bit of documentation.
```suggestion
/datum/species/proc/throw_arousal_alert(level, atom/movable/screen/alert/aroused_x/arousal_alert, mob/living/carbon/human/targeted_human)
```